test/cloudformation/timestream_cloudformation.yaml

```
@@ -6,7 +6,10 @@ Parameters:
     Description: "Timestream database to forward test metrics to"
   TableName:
     Type: String
-    Description: "Timestream table to forward test metrics to"
+    Description: "Timestream table to forward scale test metrics to "
+  ResourceTableName:
```

```suggestion
  SweeperTableName:
```
Just because it refers to the sweeper cleanup